.rubocop_todo.yml

```diff
@@ -10,7 +10,9 @@ Metrics/AbcSize:
 Metrics/CyclomaticComplexity:
   Exclude:
   - lib/autoproj/cli/ci.rb
+  - lib/autoproj/cli/main_ci.rb
 
 Metrics/PerceivedComplexity:
   Exclude:
   - lib/autoproj/cli/ci.rb
+  - lib/autoproj/cli/main_ci.rb
```

Comment:
Well ... Those exclusions were added when I converted the package to rubocop. Don't *add* anything to them. The only allowed operation is removing.

I.e. It's not that the complexity cops are optional, it's that they would have blocked converting to rubocop.